### PR TITLE
Generate more idomatic screenshots.html file when using screengrab.

### PR DESCRIPTION
**Pull request by vanniktech**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid (this actually threw some exceptions already before)
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

Fixing #10878

### Description

I've made some changes to the template to fix the above issue.

The generated output for the html site looks like actual code now that you'd write too and is also correctly indented.

**Before**

screen shot 2018-02-09 at 14 14 34

**After**

screen shot 2018-02-09 at 14 13 39

This allows the file to be correctly checked in into the repository and then having a minimal diff when adding or changing some more languages or device types.

I've updated both the templating for iOS & Android. I'll write a few more things in code that I changed.

I'd advise to review this PR by disabling whitespace changes (adding `?w=1` to the url).

### CODE REVIEW

File: screengrab/lib/screengrab/page.html.erb [hunk 1]
```diff
@@ -64,117 +64,99 @@
       }
     </style>
   </head>
-  <body>
-    <% divide_size_by = 5.0 %>
-    <% max_width = 180 %>
-    <% image_counter = 0 %>
+  <body><% image_counter = 0 %><% @data.each do |language, content| %>
```
Comment (vanniktech) [inline screengrab/lib/screengrab/page.html.erb]: Had to put all of those `<% .. %>` on the same line in order to not generate new lines. Is there any other way around this?

File: screengrab/lib/screengrab/page.html.erb [hunk 2]
```diff
@@ -64,117 +64,99 @@
...
+    <table><% content.each do |device_name, screens| %>
+      <tr>
+        <th colspan="<%= screens.count %>">
+          <span class="deviceName"><%= device_name %></span>
+        </th>
+      </tr>
+      <tr><% screens.each do |screen_path| %><% next if screen_path.include?"_framed.png" %>
+        <td><% image_counter += 1 %>
+          <a href="<%= screen_path %>" target="_blank" class="screenshotLink">
+            <img class="screenshot" src="<%= screen_path %>" style="width: 100%;" alt="<%= language %> <%= device_name %>" data-counter="<%= image_counter %>">
```
Comment (vanniktech) [inline screengrab/lib/screengrab/page.html.erb]: Inlined this entire image declaration into one line and also inlined the `style` variable.

File: screengrab/lib/screengrab/page.html.erb [hunk 3]
```diff
@@ -64,117 +64,99 @@
       }
     </style>
   </head>
-  <body>
-    <% divide_size_by = 5.0 %>
```
Comment (vanniktech) [inline screengrab/lib/screengrab/page.html.erb]: Removed this variable as it was unused.

File: screengrab/lib/screengrab/page.html.erb [hunk 4]
```diff
@@ -64,117 +64,99 @@
...
-    <% max_width = 180 %>
```
Comment (vanniktech) [inline screengrab/lib/screengrab/page.html.erb]: Removed this variable as it was unused.

File: screengrab/lib/screengrab/page.html.erb [hunk 5]
```diff
@@ -64,117 +64,99 @@
...
-            imageDisplay.dataset.counter = img.dataset.counter;
+          imageDisplay.src             = img.src;
+          imageDisplay.alt             = img.alt;
+          imageDisplay.dataset.counter = img.dataset.counter;
 
-            imageInfo.innerHTML          = '<h3>'+img.alt+'</h3>';
-            imageInfo.innerHTML         += img.src.split("/").pop();
-            imageInfo.innerHTML         += '<br />'+tmpImg.height+'&times;'+tmpImg.width+'px';
+          imageInfo.innerHTML          = '<h3>'+img.alt+'</h3>';
+          imageInfo.innerHTML         += decodeURI(img.src.split("/").pop());
```
Comment (vanniktech) [inline screengrab/lib/screengrab/page.html.erb]: Added `decodeURI` which was there in the iOS part. Correct me if this is wrong or whether it should be removed on both sides.

**Comment by KrauseFx** [comment]
Thanks for taking the time @vanniktech, this looks like various HTML changes, if you tested this locally and can confirm this works as expected, I'm happy to merge it 👍 

I'd appreciate if you were available for any follow-up changes that might be needed based on reports from users on this one

**Comment by vanniktech** [comment]
Yes I've tested this locally by using the Gemfile approach. Also, the changes are pretty small when reviewed like [this](https://github.com/fastlane/fastlane/pull/11811/files?w=1).

Just mention me in any issues regarding this and I'll reply.

**Review by KrauseFx — APPROVED** [review]
Awesome, thanks for confirming 👍

**Pull request merged**